public/locales/en/common.json

```
@@ -780,6 +780,15 @@
         "passed": "Passed",
         "failed": "Failed"
     },
+    "openwrt": {
+        "uptime": "Uptime",
+        "uptimeValue": "{{days}}d {{hours}}h {{minutes}}m {{seconds}}s",
+        "cpuLoad": "CPU Load Avg (5 mins)",
```

```suggestion
        "cpuLoad": "CPU Load Avg",
```

Or at most `(5m)`
people complain a lot about long labels around here =)

---

I stuck to `(5m)`, openwrt returns 3 values so I'd prefer to note which one if possible